conflictlog/memcached.go

```diff
@@ -164,6 +197,15 @@ func (m *MemcachedConn) setMeta(key string, body []byte, collId uint32, dataType
 			err = ErrUnknownCollection
 			return
 		}
+		if rsp.Status == gomemcached.NOT_MY_VBUCKET {
+			m.logger.Infof("got NOT_MY_BUCKET bucketName=%s", m.bucketName)
```

Comment:
"not_my_Vbucket"

Reply:
Did not understand it. Are you referring to the type `NOT_MY_BUCKET`?

Reply:
Line 200 is `gomemcached.NOT_MY_VBUCKET` but line 201 the log is syaing `NOT_MY_BUCKET`

Reply:
The typo is fixed